.env.eon

```diff
@@ -15,7 +15,7 @@ NEXT_PUBLIC_IS_TESTNET=false
 NEXT_PUBLIC_API_HOST=eon-explorer.horizenlabs.io
 
 # HOMEPAGE
-NEXT_PUBLIC_HOMEPAGE_CHARTS=['daily_txs', 'coin_price', 'market_cap', 'tvl']
+NEXT_PUBLIC_HOMEPAGE_CHARTS=['daily_txs', 'coin_price']
```

Comment:
@davehorizen did you also recommend hiding the current `coin_price`?

Reply:
@yurikoinaba @jmontesdeocanuez if its source is the `v2` endpoint `/api/v2/stats/charts/market` I will hide also the `coin_price` for now. Otherwise we can check if it's retrieved by a reliable source.